crates/primitives/src/chain/spec.rs

```diff
@@ -258,8 +259,143 @@ impl BaseFeeParams {
             elasticity_multiplier: EIP1559_DEFAULT_ELASTICITY_MULTIPLIER,
         }
     }
+
+    /// Get the base fee parameters for optimism goerli
+    #[cfg(feature = "optimism")]
+    pub const fn optimism_goerli() -> BaseFeeParams {
+        BaseFeeParams { max_change_denominator: 50, elasticity_multiplier: 10 }
+    }
+
+    /// Get the base fee parameters for optimism mainnet
+    #[cfg(feature = "optimism")]
+    pub const fn optimism() -> BaseFeeParams {
+        BaseFeeParams { max_change_denominator: 50, elasticity_multiplier: 6 }
+    }
 }
 
+/// The Optimism Goerli spec
```

Comment:
nit pull these above next to the other genesis

Reply:
Addressed in https://github.com/anton-rs/op-reth/pull/161